ocaml/xenopsd/pvs/pvs_setup.md

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
A good location for this file would be `docs/content/xenopsd/design`. It needs a preample like this:
```
+++
title = "PVS Proxy OVS Rules"
+++
```
Then, once merged, it will pop up under the menu here: https://xapi-project.github.io/new-docs/xenopsd/design.